sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -450,57 +407,6 @@ def test_predict_interval(
             with pytest.raises(NotImplementedError, match="prediction intervals"):
                 estimator_instance.predict_interval(fh_int_oos, coverage=coverage)
 
-    def _check_predict_quantiles(self, pred_quantiles, y_train, fh, alpha):
```

Comment:
can you comment, are you changing these methods?

Reply:
They are extended to handle multiindex data. The behavior of these methods in `TestAllGlobalForecasters` is actully the same.